lib/plucky/criteria_hash.rb

```diff
@@ -164,14 +166,12 @@ def modifier_merge(hash, value)
 
     # Private
     def merge_values_into_array(value, other_value)
-      array(value).concat(array(other_value)).uniq
+      array(value).concat(array(other_value)).flatten.uniq
```

Comment:
Interesting. Never thought of that.  We should have a test against that case.

Best

Scott

> On Feb 4, 2016, at 12:41 PM, Chris Heald <notifications@github.com> wrote:
> 
> In lib/plucky/criteria_hash.rb:
> 
> > @@ -164,14 +166,12 @@ def modifier_merge(hash, value)
> >  
> >      # Private
> >      def merge_values_into_array(value, other_value)
> > -      array(value).concat(array(other_value)).uniq
> > +      array(value).concat(array(other_value)).flatten.uniq
> I haven't unwound how this is used, but would this break docs storing nested array values?
> 
> —
> Reply to this email directly or view it on GitHub.
>